### DISCUSSION TITLE
I would like to know the printing condition of `zfs send` prints the progress always.

### DISCUSSION THREAD

**Discussion by mikanmi** [openzfs/zfs#13966] [Q&A] [answered]
I would like to show the progress on a backup with `zfs send` and `zfs recv`.

I am developing [Elephant Backup](https://github.com/patineboot/elephant-backup), which is backup CLI to archive ZFS filesystems.
Elephant Backup uses the `zfs send` and the `zfs recv` command from own itself Javascript language.

e.g., Elephant Backup's usage of the zfs commands.
- `zfs send -Rw   -I storage.pool@elephant-2022-08-06-123351 storage.pool@elephant-2022-09-28-123351`
- `zfs recv -F -d -x mountpoint backup.pool/storage.pool`

I would like to show the progress on a backup without the `pv` command.
I found that sometimes, `zfs send` prints the progress on a backup on the standard error or not.
But I do not know the printing condition of `zfs send`.

I would like to know the printing condition of `zfs send` prints the progress always.

**Comment by rincebrain** [comment, marked as answer]
`zfs send -v`, maybe?

**Reply by mikanmi** [reply]
Thanks, `zfs send -v` prints the progress on the standard error.
